implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogMeterRegistry.java

```diff
@@ -230,6 +344,50 @@ private void addToMetadataList(Map<String, DatadogMetricMetadata> metadata, Mete
         }
     }
 
+    // VisibleForTesting
+    void writeMetricViaStatsd(Meter.Id id, @Nullable String suffix, double value, Statistic statistic,
+            @Nullable String overrideBaseUnit) {
+        if (statsDClient == null) {
+            return;
+        }
+
+        Meter.Id fullId = id;
+        if (suffix != null)
+            fullId = idWithSuffix(id, suffix);
+
+        Iterable<Tag> tags = getConventionTags(fullId);
+
+        List<String> tagsList = tags.iterator().hasNext() ? stream(tags.spliterator(), false)
+            .map(t -> "\"" + escapeJson(t.getKey()) + ":" + escapeJson(t.getValue()) + "\"")
+            .collect(toList()) : new ArrayList<String>();
+        if (config.hostTag() != null && !Objects.equals(config.hostTag(), "")) {
+            tagsList.add("host:" + config.hostTag());
+        }
+        String[] tagsArray = new String[tagsList.size()];
+        tagsList.toArray(tagsArray);
+
+        String metricName = getConventionName(fullId);
+
+        // Create type attribute
+        switch (fullId.getType()) {
+            case COUNTER:
+                statsDClient.count(metricName, value, 1.0, tagsArray);
+                break;
+            case GAUGE:
+                statsDClient.gauge(metricName, value, 1.0, tagsArray);
+                break;
+            case LONG_TASK_TIMER:
```

Comment:
These 3 cases are not supported in DataDog?

Reply:
this is a fall-through clause to `distribution` type. Timer's are typically reported as distributions so you can get min/max/avg/p##..

Reply:
OK I see, thank you